packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -198,7 +200,7 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
     isCctp,
     isDepositMode,
     parentChainProvider,
-    token?.address,
+    token,
```

Comment:
We should avoid passing objects, and extract only the required data points in the dependency array - `token?.address`, `token?.parentBridgeAddress` and `token?.childBridgeAddress`

Reply:
Passing object is fine if the reference is stable